Optimobile/AppDelegate.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  AppDelegate.swift
+//  Optimobile
+//
+//  Created by Barak Ben Hur on 04/04/2022.
+//
+
+import UIKit
+import KumulosSDK
+
+@main
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        let builder = KSConfigBuilder(apiKey: "2b16f7c7-6f0e-46c1-979f-1c13b5fd8a0f", secretKey: "Xh62vshcS15xJCjSHiQ2dcPvfXHVirKdT+Vg").enableInAppMessaging(inAppConsentStrategy: InAppConsentStrategy.AutoEnroll).setPushOpenedHandler(pushOpenedHandlerBlock: { (notification : KSPushNotification) -> Void in
+            //- Inspect notification data and do work.
+            if let action = notification.actionIdentifier {
+                print("User pressed an action button.")
+                print(action)
+                print(notification.data)
+            } else {
+                print("Just an open event.")
+            }
+        }).setInAppDeepLinkHandler(inAppDeepLinkHandlerBlock: { buttonPress in
+            let deepLink = buttonPress.deepLinkData
+            let messageData = buttonPress.messageData
+            print(deepLink)
+            print(messageData ?? "")
+            // TODO: Inspect the deep link & message data to perform relevant action
+        }).setPushOpenedHandler(pushOpenedHandlerBlock: { (notification : KSPushNotification) -> Void in
+            print(notification)
+        }).enableDeepLinking({ (resolution) in
+            print("Deep link resolution result: \(resolution)")
+        }).enableCrash()
```

Comment:
Crash features should be stripped out, is plan to retain the KSConfig builder approach or to call methods on the singleton?